katdal/concatdata.py

```diff
@@ -267,18 +302,29 @@ class ConcatenatedSensorCache(SensorCache):
         be applied to sensor data (this can be disabled on data retrieval)
 
     """
+
     def __init__(self, caches, keep=None):
         self.caches = caches
         # Collect the names and types of all actual and virtual sensors in caches, as well as properties
```

Comment:
This comment needs to be updated given that actual now collects the sensor_data objects themselves.